### PR TITLE
Rahulbcn27 GitHub actions 1

### PR DESCRIPTION
**Pull request by rahulbcn27**
Created Github-Actions build.yml

### CODE REVIEW

**Comment by rahulbcn27** [comment]
@rohityadavcloud Please review.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Build-Check
```
Comment (rohityadavcloud) [inline .github/workflows/build.yml:1]: @rahulbcn27 pl add license

**Comment by rohityadavcloud** [comment]
@rahulbcn27 could you review https://github.com/apache/cloudstack-terraform-provider#testing-the-provider

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  build:
+    runs-on: ubuntu-22.04
+    
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.16.x
```
Comment (rohityadavcloud) [inline .github/workflows/build.yml:19]: Should we bump the version to latest?

**Review by rohityadavcloud — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (rahulbcn27) [reply .github/workflows/build.yml:18]: license added

File: .github/workflows/build.yml (hunk 2 above)
Comment (rahulbcn27) [reply .github/workflows/build.yml]: version updated to 1.19.x

**Comment by rohityadavcloud** [comment]
For basic build and unit tests this is good. However, we need to explore if some basic CI/CD using https://github.com/apache/cloudstack-terraform-provider#testing-the-provider (simulator) can be achieved as well. I'll merge this but explore further with @rahulbcn27 about this.

**Pull request merged**